com.sap.cloud.lm.sl.mta/src/test/resources/com/sap/cloud/lm/sl/mta/resolvers/v3_1/platform-1.json

```diff
@@ -0,0 +1,41 @@
+[
```

Comment:
Can we extract the file on level /src/test/resources/com/sap/cloud/lm/sl/mta/resolvers and re-use it both for v2 and v3 resolvers?